### ISSUE TITLE
[mysql] log query string when mysql_session error

### ISSUE THREAD

**Issue by BohuTANG** [datafuselabs/datafuse#985]
**Summary**

```
[2021-07-04T11:16:08Z ERROR fuse_query::servers::mysql::mysql_session] Unexpected error occurred during query execution: Code: 1002, displayText = peer terminated connection.
    
       0: common_exception::exception::ErrorCode::from_std_error
                 at common/exception/src/exception.rs:328:65
          <common_exception::exception::ErrorCode as core::convert::From<std::io::error::Error>>::from
                 at common/exception/src/exception.rs:312:9
       1: <core::result::Result<T,F> as core::ops::try_trait::FromResidual<core::result::Result<core::convert::Infallible,E>>>::from_residual
                 at /rustc/657bc01888e6297257655585f9c475a0801db6d2/library/core/src/result.rs:1675:27
          msql_srv::MysqlIntermediary<B,R,W>::init
                 at /root/.cargo/git/checkouts/msql-srv-62b5153e0120bb3a/5a7ae3d/src/lib.rs:257:36
          msql_srv::MysqlIntermediary<B,R,W>::run_on
                 at /root/.cargo/git/checkouts/msql-srv-62b5153e0120bb3a/5a7ae3d/src/lib.rs:234:9
          msql_srv::MysqlIntermediary<B,std::net::tcp::TcpStream,std::net::tcp::TcpStream>::run_on_tcp
                 at /root/.cargo/git/checkouts/msql-srv-62b5153e0120bb3a/5a7ae3d/src/lib.rs:203:9
       2: <fuse_query::servers::mysql::mysql_session::Session as fuse_query::servers::abortable::AbortableService<tokio::net::tcp::stream::TcpStream,()>>::start::{{closure}}::{{closure}}
                 at fusequery/query/src/servers/mysql/mysql_session.rs:90:17
          std::sys_common::backtrace::__rust_begin_short_backtrace
                 at /rustc/657bc01888e6297257655585f9c475a0801db6d2/library/std/src/sys_common/backtrace.rs:125:18
       3: std::thread::Builder::spawn_unchecked::{{closure}}::{{closure}}
                 at /rustc/657bc01888e6297257655585f9c475a0801db6d2/library/std/src/thread/mod.rs:481:17
          <std::panic::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
                 at /rustc/657bc01888e6297257655585f9c475a0801db6d2/library/std/src/panic.rs:347:9
          std::panicking::try::do_call
                 at /rustc/657bc01888e6297257655585f9c475a0801db6d2/library/std/src/panicking.rs:401:40
          std::panicking::try
                 at /rustc/657bc01888e6297257655585f9c475a0801db6d2/library/std/src/panicking.rs:365:19
          std::panic::catch_unwind
                 at /rustc/657bc01888e6297257655585f9c475a0801db6d2/library/std/src/panic.rs:434:14
          std::thread::Builder::spawn_unchecked::{{closure}}
                 at /rustc/657bc01888e6297257655585f9c475a0801db6d2/library/std/src/thread/mod.rs:480:30
          core::ops::function::FnOnce::call_once{{vtable.shim}}
                 at /rustc/657bc01888e6297257655585f9c475a0801db6d2/library/core/src/ops/function.rs:227:5
       4: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
                 at /rustc/657bc01888e6297257655585f9c475a0801db6d2/library/alloc/src/boxed.rs:1575:9
          <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
                 at /rustc/657bc01888e6297257655585f9c475a0801db6d2/library/alloc/src/boxed.rs:1575:9
          std::sys::unix::thread::Thread::new::thread_start
                 at /rustc/657bc01888e6297257655585f9c475a0801db6d2/library/std/src/sys/unix/thread.rs:71:17
       5: start_thread
       6: clone
```

**Comment by dust1** [comment]
Sorry to disturb.
I am executing some wrong statements like `select id from unknow_table`, the place where the exception is thrown is `MySqlError`. Will it be thrown as long as it is an exception during query?
If only mysql_session error,How can I reproduce this exception? :(

**Comment by PsiACE** [comment]
> If only mysql_session error,How can I reproduce this exception? :(

like OOM in https://github.com/datafuselabs/datafuse/issues/1274

SELECT number FROM numbers_mt(10000000000) ORDER BY number DESC LIMIT 100

**Comment by dust1** [comment]
> > If only mysql_session error,How can I reproduce this exception? :(
> 
> like OOM in #1274
> 
> SELECT number FROM numbers_mt(10000000000) ORDER BY number DESC LIMIT 100

thanks. :)

**Comment by zhang2014** [comment]
fixed in #1353
